lib/inc/internal/facts/linux/os_cisco.hpp

```diff
@@ -57,7 +57,17 @@ namespace facter { namespace facts { namespace linux {
                 return value;
             }
             auto val = _release_info.find("ID_LIKE");
-            return (val != _release_info.end()) ? val->second : std::string();
+            if (val != _release_info.end()) {
+                auto family = val->second;
```

Comment:
Nit pick: this makes a copy of the string, which is fine when it doesn't contain a space (we were copying it when returning it anyway), but in the case where it does contain a space, the copy doesn't need to be made.

Using `auto& family = val->second;` creates a reference to the value, which we'll only copy if it doesn't contain a space.